docs/design/design-patterns/README.md

```diff
@@ -0,0 +1,8 @@
+# Design Patterns
+
+The design patterns section recommends patterns of software and architecture design.
+We want to be very clear as to not replicating content that already exists but instead centralizing vetted sources to use into the playbook with suggestions, guidance and learnings based on our firsthand experiences.
+
+## Subsections
+
+* [Data Fundamentals](data-fundamentals/README.md)
```

Comment:
Would perhaps suggest renaming to "Data Heavy Design Guidance" or "Data Heavy Solution Guidance" or "Data Heavy Pattern Guidance". This would require renaming the md file as well as editing the title of the section within the md file.